openpype/hosts/max/startup/startup.py

```diff
@@ -1,6 +1,20 @@
 # -*- coding: utf-8 -*-
-from openpype.hosts.max.api import MaxHost
-from openpype.pipeline import install_host
+import os
+import sys
+
+# this might happen in some 3dsmax version where PYTHONPATH isn't added
+# to sys.path automatically
+try:
+    from openpype.hosts.max.api import MaxHost
+    from openpype.pipeline import install_host
+except (ImportError, ModuleNotFoundError):
+
+    for path in os.environ["PYTHONPATH"].split(os.pathsep):
+        if path and path not in sys.path:
+            sys.path.append(path)
+
+    from openpype.hosts.max.api import MaxHost
+    from openpype.pipeline import install_host
```

Comment:
Could be simplified to
```suggestion
for path in os.environ["PYTHONPATH"].split(os.pathsep):
    if path and path not in sys.path:
        sys.path.append(path)

from openpype.hosts.max.api import MaxHost
from openpype.pipeline import install_host
```